### PR TITLE
chore: move to GitHub container registry

### PR DESCRIPTION
**Pull request by smira**
Username & registry updates, new docker login.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by rsmitty — APPROVED** [review]

**Review by andrewrynhard — COMMENTED** [review]
nit: `ghcr-username` to `ghcr_username` so that it aligns with the convention.

File: .drone.yml [hunk 1]
```diff
@@ -47,11 +47,11 @@ steps:
     pull: always
     environment:
       DOCKER_USERNAME:
-        from_secret: docker_username
+        from_secret: ghcr-username
```
Comment (andrewrynhard) [inline .drone.yml:50]: nit: `ghcr-username` to `ghcr_username` so that it aligns with the convention.

File: .drone.yml [hunk 2]
```diff
@@ -47,11 +47,11 @@ steps:
...
       DOCKER_PASSWORD:
-        from_secret: docker_password
+        from_secret: ghcr-token
```
Comment (andrewrynhard) [inline .drone.yml:52]: nit: `ghcr-token` to `ghcr_token` so that it aligns with the convention.

**Review by andrewrynhard — APPROVED** [review]

File: .drone.yml [hunk 3]
```diff
@@ -47,11 +47,11 @@ steps:
...
     commands:
-      - docker login --username "$${DOCKER_USERNAME}" --password "$${DOCKER_PASSWORD}"
+      - docker login ghcr.io --username "$${DOCKER_USERNAME}" --password "$${DOCKER_PASSWORD}"
```
[the remark this answers is not in the document]
Comment (smira) [reply .drone.yml:54]: I can change that, I just wanted to minimize the changes... just a second

**Pull request merged**